src/couch_mrview/src/couch_mrview_index.erl

```diff
@@ -207,3 +217,104 @@ index_file_exists(State) ->
     } = State,
     IndexFName = couch_mrview_util:index_file(DbName, Sig),
     filelib:is_file(IndexFName).
+
+
+verify_index_exists(Props) ->
+    ShardDbName = couch_mrview_util:get_value_from_options(
```

Comment:
Users can write to `_local/` docs. What prevents malicious user from specifying someone else `dbname`? What would happen if they would do it?

Reply:
Can malicious user have right to work on other's database?